syntax = "proto3";

message ProcessInfo {
  uint32 pid = 1;
  string name = 2;
  double cpu_usage = 3;
  string full_path = 4;
  string command_line = 5;
  bool is_64_bit = 6;
}

